10up/brightcove-video-connect
# Adjust position of media details in the modal

### Description of the Change

This change adjusts the position of media details in the modal.

### Alternate Designs

### Benefits

If the user scrolls down and selects a video, it's necessary to scroll back to see the media details. This change adjusts the position of media details to avoid that.

### Possible Drawbacks

### Verification Process

1. Add the Brightcove block and edit it
2. Scroll down and select a video
3. The media details should appear in a near position

### Checklist:






### Applicable Issues



### Changelog Entry
